src/Microsoft.AspNet.Mvc.Core/Formatters/JsonOutputFormatter.cs

```diff
@@ -16,13 +17,23 @@ public class JsonOutputFormatter : OutputFormatter, IJsonOutputFormatter
         private JsonSerializerSettings _serializerSettings;
 
         public JsonOutputFormatter()
```

Comment:
Make this `internal` if it's now used only in tests (and add comment indicating that's why it exists at all :smile_cat:).  Doubt we want anyone using this one since it ignores `MvcOptions`.  Same for similar constructor overload in `JsonInputFormatter`.